src/modules/cmdpal/WindowsCommandPalette/PageViewModel.cs

```diff
@@ -17,7 +16,20 @@ public class PageViewModel
     // public IPage PageAction { get => pageAction; set => pageAction = value; }
     public ActionViewModel Command { get; }
 
-    public Color AccentColor => PageAction.AccentColor;
+    public Windows.UI.Color AccentColor
```

Comment:
Forgot this was the old app for a second... 😅

Reply:
Oh yea that codebase is in FULL ON "yolo as long as it still kinda runs" mode